Watchman.Discord/Areas/Administration/Controllers/AdministrationController.cs

```diff
@@ -126,7 +126,12 @@ public async Task GetSafeUsers(SafeUsersCommand safeUsersCommand, Contexts conte
                 values.Add(new KeyValuePair<string, string>($"{user.Mention} -", user.JoinedServerAt()?.ToLocalTimeString() ?? "nieznana data"));
             }
             var messagesService = this._messagesServiceFactory.Create(contexts);
-            var safeUsers = new Dictionary<string, Dictionary<string, string>>()
+            if (!values.Any())
+            {
+                await messagesService.SendEmbedMessage("Zaufani użytkownicy", "Serwer nie posiada zaufanych użytkowników", new Dictionary<string, string>());
```

Comment:
kurde trzeba będzie rozbudować możliwości Responses bo coraz więcej mamy odpowiedzi na sztywno w kodzie
ale to już później